File: api-examples/cat/search/js/spatial_search_linestring.js
var settings = {
  "url": "https://<catalogue-server-ip>/catalogue/v1/search?geometry=linestring(18.56,73.83,18.49,73.8)&relation=intersects",
  "method": "GET",
  "timeout": 0,
};

$.ajax(settings).done(function (response) {
  console.log(response);
});

File: api-examples/cat/search/java/spatial_search_linestring.java
OkHttpClient client = new OkHttpClient().newBuilder()
  .build();
Request request = new Request.Builder()
  .url("https://<catalogue-server-ip>/catalogue/v1/search?geometry=linestring(18.56,73.83,18.49,73.8)&relation=intersects")
  .method("GET", null)
  .build();
Response response = client.newCall(request).execute();
